source/core/parameters_multiphysics.cc

```diff
@@ -67,6 +67,12 @@ Parameters::Multiphysics::declare_parameters(ParameterHandler &prm)
                       Patterns::Bool(),
                       "VOF calculation <true|false>");
 
+    prm.declare_entry(
+      "use average velocity field",
+      "false",
+      Patterns::Bool(),
+      "Use the average velocity field instead of the present velocity field <true|false>");
```

Comment:
```suggestion
      "Use the average velocity field in subphysics, instead of the present velocity field <true|false>"
      "Note: currently implemented only for heat transfer");
```